web/modules/custom/joinup_core/joinup_core.behat.inc

```diff
@@ -17,9 +17,15 @@ class JoinupCoreSubContext extends DrupalSubContextBase implements DrupalSubCont
   /**
    * Asserts that only the expected chips are shown in the page.
    *
+   * A "chip" is a Material Design element that is used in dropdowns with multi-
+   * select to show which items have been selected when the dropdown is
+   * collapsed.
+   *
    * @param \Behat\Gherkin\Node\TableNode $table
    *   The table containing the expected chip labels.
    *
+   * @see https://material.io/guidelines/components/chips.html
```

Comment:
You should put the french page as well https://material.io/guidelines/components/fr/fries.html :P